starknet_py/serialization/factory.py

```diff
@@ -3,9 +3,9 @@
 from collections import OrderedDict
 from typing import Dict, List, Union
 
-from starknet_py.abi.model import Abi
-from starknet_py.abi.v1.model import Abi as AbiV1
-from starknet_py.abi.v2.model import Abi as AbiV2
+from starknet_py.abi.v0 import Abi
```

Comment:
nit: Also in other files
```suggestion
from starknet_py.abi.v0 import Abi as AbiV0
```

Reply:
I will add for files where we import more than one `starknet_py.abi.*`